protected/humhub/modules/content/widgets/stream/views/wallStreamEntryHeader.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+use humhub\libs\Html;
+use humhub\modules\content\components\ContentActiveRecord;
+use humhub\modules\content\widgets\ArchivedIcon;
+use humhub\modules\content\widgets\stream\WallStreamEntryOptions;
+use humhub\modules\content\widgets\UpdatedIcon;
+use humhub\modules\content\widgets\VisibilityIcon;
+use humhub\modules\content\widgets\WallEntryControls;
+use humhub\modules\space\models\Space;
+use humhub\modules\user\models\User;
+use humhub\modules\space\widgets\Image as SpaceImage;
+use humhub\modules\ui\icon\widgets\Icon;
+use humhub\modules\ui\view\components\View;
+use humhub\widgets\TimeAgo;
+
+/* @var $this View */
+/* @var $model ContentActiveRecord */
+/* @var $headImage string */
+/* @var $permaLink string */
+/* @var $title string */
+/* @var $renderOptions WallStreamEntryOptions */
+
+$container = $model->content->container;
+?>
+
+<div class="stream-entry-icon-list">
+    <?php if ($model->content->isArchived()) : ?>
+        <?= ArchivedIcon::getByModel($model) ?>
+    <?php elseif ($renderOptions->isPinned($model)) : ?>
+        <?= Icon::get('map-pin', ['htmlOptions' => ['class' => 'icon-pin tt', 'title' => Yii::t('ContentModule.base', 'Pinned')]]) ?>
+    <?php endif; ?>
+</div>
+
+<!-- since v1.2 -->
+<div class="stream-entry-loader"></div>
+
+<!-- start: show wall entry options -->
+<?php if (!$renderOptions->isControlsMenuDisabled()) : ?>
+    <?= WallEntryControls::widget(['renderOptions' => $renderOptions, 'object' => $model, 'wallEntryWidget' => $this->context]) ?>
+<?php endif; ?>
+<!-- end: show wall entry options -->
+
+
+<div class="wall-entry-header-image">
+    <?= $headImage ?>
+</div>
+
+<div class="wall-entry-header-info media-body">
+
+    <div class="media-heading">
+        <?= $title ?>
```

Comment:
How is the HTML encoding handled?

Reply:
See: 
- https://github.com/humhub/humhub/pull/4397/files#diff-88090f8dfe8ba919be8c3443a5e4a53fR34
- https://github.com/humhub/humhub/pull/4397/files#diff-88090f8dfe8ba919be8c3443a5e4a53fR78